File: src/ERP/acme.atena.domain/DTO/Seguranca/PermissaoUsuario.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.atena.domain.DTO.Seguranca
{
    public class PermissaoUsuario : AbstractEntity
    {
        public string Acesso { get; set; }

        public Guid TelaId { get; set; }

        public Tela Tela { get; set; }
        public Guid UsuarioId { get; set; }
        public Guid PermissaoId { get; set; }

        public virtual Usuario Usuario{get;set;}
        public virtual Permissao Permissao { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/security/permissao-usuario.ts
import { Permissao } from './permissao';
import { UsuarioResponse } from './usuario';
import { AbstractEntity } from "../abstract-entity";
import { Tela } from './tela';

export class PermissaoUsuario extends AbstractEntity{

  public  acesso: string;

  public usuarioId : string;
  public permissaoId : string;
  public telaId: string;

  public usuario : UsuarioResponse;
  public permissao: Permissao = new Permissao();
  public tela: Tela = new Tela();
}
